get_all_data/common_field_transforms.py

```diff
@@ -0,0 +1,40 @@
+import dateutil.parser
+import datetime
+
+def convert_to_header(field):
```

Comment:
Should we move these functions into Pytx?  I'd imagine most anyone will want this functionality.